WebJobs.Script.proj

```diff
@@ -6,6 +6,11 @@
     <SkipStrongNamesExe>packages\Microsoft.Web.SkipStrongNames.1.0.0\tools\SkipStrongNames.exe</SkipStrongNamesExe>
     <SkipStrongNamesXml>tools\SkipStrongNames.xml</SkipStrongNamesXml>
     <PublishPath Condition=" '$(PublishPath)' == '' ">bin</PublishPath>
+    <BuildNumber Condition=" '$(BuildNumber)' == '' ">0</BuildNumber>
+    <SiteExtensionVersion>1.0.$(BuildNumber)</SiteExtensionVersion>
```

Comment:
I meant to ask @davidebbo how he wants to handle this versioning. This will result in a public version that jumps around quite a bit, versus something that only increments when we do a release.

To get the latter, in other projects I've simply added a non-auto version param to the CI, which I manually increment when we want to bump the version.